hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -182,7 +182,8 @@
     PROJECT_SERVICE_NOT_FOUND(71, "service was not found.", Response.Status.BAD_REQUEST),
     QUOTA_REQUEST_NOT_COMPLETE(72, "Please specify both " + "namespace and space quota.", Response.Status.BAD_REQUEST),
     RESERVED_PROJECT_NAME(73, "Not allowed - reserved project name, pick another project name.",
-        Response.Status.BAD_REQUEST);
+        Response.Status.BAD_REQUEST),
+    PROJECT_ANACONDA_ENABLE_ERROR(71, "Failed to enable conda.", Response.Status.INTERNAL_SERVER_ERROR);
```

Comment:
@ErmiasG how did that go through testing mvn tests? It has the same code as `PROJECT_SERVICE_NOT_FOUND`

Reply:
Didn't